test/unit/agent_metadata/test_aws_ec2_instance.py

```diff
@@ -41,6 +41,10 @@ def around(self):
                     httpretty.GET,
                     EC2_HOST_INSTANCE_TYPE_URI,
                     body="testHostType")
+                httpretty.register_uri(
+                    httpretty.GET,
+                    EC2_API_TOKEN_URI,
+                    body="PARIOq_FXbIyL0maE9RcmrsyWtylvFh1ZDt0NrRUyNxeV1-DlpFpA==")
```

Comment:
not sure you need to provide a real token here, it may trigger some security scripts that search the code for those. Could you just put a fake string here?

Reply:
It's not real, it's based on a real one, but then I updates random characters from it.

Reply:
But I agree it still can trigger some security scripts, I will update.

Reply:
Just updated it, thanks.